test/integration/roles/test_gcp_healthcheck/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+# GCP Healthcheck Integration Tests.
+# TODO(supertom): in progress, NOT WORKING
+######
+- name: param check
+  gcp_healthcheck:
+    service_account_email: "{{ service_account_email }}"
+    credentials_file: "{{ credentials_file }}"
+    project_id: "{{ project_id }}"
+    healthcheck_name: "{{ http_healthcheck }}"
+    healthcheck_type: HTTP
+    host_header: my-host
+    request_path: /hc
+    check_interval: 10
+    timeout: 30
+    unhealthy_threshold: 2
+    healthy_threshold: 1
+    state: present
+  ignore_errors: True
+  register: result
+- name: check interval
+  assert:
+    that:
+      'result.msg == "timeout (30) is greater than check_interval (10)"'
+- name: create "{{ http_healthcheck }}"
+  gcp_healthcheck:
+    service_account_email: "{{ service_account_email }}"
+    credentials_file: "{{ credentials_file }}"
+    project_id: "{{ project_id }}"
+    healthcheck_name: "{{ http_healthcheck }}"
+    healthcheck_type: HTTP
+    host_header: my-host
+    request_path: /hc
+    check_interval: 5
+    timeout: 5
+    unhealthy_threshold: 2
+    healthy_threshold: 1
+    state: present
+  register: result
+- name: assert create
+  assert:
+    that:
+      - 'result.state == "present"'
+      - 'result.changed'
+- name: create minimum "{{ https_healthcheck }}"
+  gcp_healthcheck:
+    service_account_email: "{{ service_account_email }}"
+    credentials_file: "{{ credentials_file }}"
+    project_id: "{{ project_id }}"
+    healthcheck_name: "{{ https_healthcheck }}"
+    healthcheck_type: HTTPS
+    state: present
+  register: result
+- name: assert create
+  assert:
+    that:
+      - 'result.state == "present"'
+      - 'result.changed'
```

Comment:
Thanks for the request.  This unearthed a bug, which is now fixed.  Added some additional tests.